libs/api/domains/education/src/lib/education.utils.ts

```diff
@@ -0,0 +1,8 @@
+export const getYearInterval = (dates: string[]) => {
```

Comment:
ditto on return type

Reply:
Typescript should be able to determine the return type here by itself